### PR TITLE
rn, atp

### PR DESCRIPTION
**Pull request by MerrytheManokit**
## About the PR ##
rn autocorrects to right now, atp autocorrects to at this point

## Why / Balance
I saw that rn and atp didn't autocorrect 

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
No changelog

### CODE REVIEW

**Comment by Plykiya** [comment]
atp feels pretty... uncommon

**Comment by MerrytheManokit** [comment]
> atp feels pretty... uncommon

i feel like it's just nice to have tbh

**Comment by deltanedas** [comment]
atm is very common, atp i have heard more wrt biology than as a phrase

**Comment by Tayrtahn** [comment]
I've never seen anyone use atp, but what do I know, I'm old.

> atm is very common, atp i have heard more wrt biology than as a phrase

Might be worth adding wrt though.

**Comment by MerrytheManokit** [comment]
> I've never seen anyone use atp, but what do I know, I'm old. rn is good.
> 
> > atm is very common, atp i have heard more wrt biology than as a phrase
> 
> Might be worth adding wrt though.

Whats wrt mean

**Comment by ike709** [comment]
I sure hope science and medical never have a reason to discuss adenosine triphosphate.

**Comment by deltanedas** [comment]
wrt is with regards to

**Comment by mirrorcult** [comment]
wort wort wort

**Comment by MerrytheManokit** [comment]
> wrt is with regards to

Reckon i should add it?

File: Resources/Locale/en-US/speech/speech-chatsan.ftl [hunk 1]
```diff
@@ -128,4 +128,13 @@ chatsan-word-46 = tyvm
...
+chatsan-replacement-47 = see ya
+
+chatsan-word-48 = rn
+chatsan-replacement-48 = right now
+
+chatsan-word-49 = atp
+chatsan-replacement-49 = at this point
+
+chatsan-word-50 = atm
+chatsan-replacement-50 = at the moment
```
Comment (metalgearsloth) [inline Resources/Locale/en-US/speech/speech-chatsan.ftl:140]:
```suggestion

chatsan-word-49 = atm
chatsan-replacement-49 = at the moment
```

File: Resources/Prototypes/Accents/word_replacements.yml [hunk 2]
```diff
@@ -428,6 +428,9 @@
     chatsan-word-45: chatsan-replacement-45
     chatsan-word-46: chatsan-replacement-46
     chatsan-word-47: chatsan-replacement-47
+    chatsan-word-48: chatsan-replacement-48
+    chatsan-word-49: chatsan-replacement-49
+    chatsan-word-50: chatsan-replacement-50
```
Comment (metalgearsloth) [inline Resources/Prototypes/Accents/word_replacements.yml:433]:
```suggestion
    chatsan-word-49: chatsan-replacement-49
```

**Comment by metalgearsloth** [comment]
> atp feels pretty... uncommon

agreed.

> > wrt is with regards to
> 
> Reckon i should add it?

this one I think is fine leaving out because if I fatfinger wt to wrt I don't want it adding a bunch of words.

**Pull request merged**